src/Raft.sol

```diff
@@ -29,7 +29,7 @@ contract Raft is ERC721EnumerableUpgradeable, UUPSUpgradeable, OwnableUpgradeabl
     address nextOwner,
     string memory name_,
     string memory symbol_
-  ) external initializer {
+  ) public initializer {
```

Comment:
what's the general rationale here?

Reply:
> You may also consider changing the initializer to `public` if you want the flexibility of calling that initializer from a child contract, for example.

https://forum.openzeppelin.com/t/slither-upgradeable-contract-does-not-protect-its-initialize-function/28432